cmd/indexer/root.go

```diff
@@ -204,10 +214,21 @@ func initViper() {
 }
 
 func assignVarFromViper() {
-	// flags for ethereum service
-	ethProtocol = viper.GetString(flags.EthProtocol)
-	ethHost = viper.GetString(flags.EthHost)
-	ethPort = viper.GetInt(flags.EthPort)
+	// flags for eth
+	// eth value has been passed via flag "--eth"
+	ethList := viper.GetStringSlice(flags.Eth)
+	for _, eth := range ethList {
+		u, _ := url.Parse(eth)
```

Comment:
should we handle error here? it seems better if it stop executing when url is wrong format